public/api/tasks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "tasks": [
+    {
+      "title": "Adding loading substate",
+      "purpose": "improving DX",
+      "startedOn": 1637148114,
+      "endsOn": 1639740111,
+      "status": "active"
+    }
+  ]
+}
```

Comment:
Is this file required?

Reply:
I was not assigned to any task through rds, due to which on tasks page I was not seeing any(blank) and issue was about showing loading message while API fetching is going on. So Ankush told me to mock the API. I guess we should keep it for such future cases.
What you think? @swarajpure

Reply:
Not required as a part of this PR, can be deleted